### DISCUSSION TITLE
Two Factor Authentication failed

### DISCUSSION THREAD

**Discussion by ghost** [community/community#77833] [Other Feature Feedback, Questions, & Ideas]
EN

Hello, GitHub enforced 2FA on me, and everything was fine for a while. However, in the last few days, when logging in from my computer, it prompts me with 'Verify 2FA Now.' When I click, it asks for an Authenticator code; I use Microsoft Authenticator, but entering the code results in an error. I tried to disable 2FA, but it's not possible. I attempted to proceed by adding my phone number, but Turkey is not listed. In summary, while mobile access works without issues, I can't access my GitHub account from my computer. Can you help? I tried resetting my password without success, and the reconfigure attempt cannot be canceled.

TR

Merhaba, GitHub beni zorunlu olarak 2FA kullanmaya yönlendirdi. Bir süre sorunsuz kullanıyordum, ancak son birkaç gündür bilgisayardan giriş yaparken "Verify 2FA Now" uyarısı alıyorum. Microsoft Authenticator kullanıyorum, ancak kodu girdiğimde hata alıyorum. 2FA'yı kapatmaya çalıştım, ancak buna izin verilmiyor. Telefon numaramı ekleyerek devam etmek istedim, ancak Türkiye listede bulunmuyor. Özetle, mobil giriş sorun yaratmazken bilgisayar üzerinden GitHub hesabıma erişemiyorum. Yardım edebilir misiniz? Şifremi sıfırlamaya çalıştım ancak başarılı olamadım ve reconfigure denemesi de iptal edilemiyor.

![IMG_20231204_105822.jpg](https://github.com/community/community/assets/32311900/6a86ed7e-381f-4f4f-b67b-b74f345414d2)

![IMG_20231204_124802.jpg](https://github.com/community/community/assets/32311900/7d5c93a1-76cf-4b89-9db0-f40a8f759b44)

Moderator's Edit: I've removed your private information from this post as posting identifiable information is not allowed, per our [Code of Conduct](https://github.com/community/community/blob/main/CODE_OF_CONDUCT.md).

![IMG_20231204_124816.jpg](https://github.com/community/community/assets/32311900/37d46b29-92e6-48a4-aa3e-09112e8a3f00)

**Comment by dakshithadissanayaka** [comment]
I understand that you're having trouble logging in to your GitHub account from your computer due to issues with two-factor authentication (2FA). Since mobile access is working without issues, it seems the problem is specific to your computer's setup.

Here are some steps you can try to resolve the issue:

1. **Clear Browser Cache and Cookies:** Try clearing your browser's cache and cookies to eliminate any potential conflicts or corrupted data.

2. **Try a Different Browser:** If the issue persists, try using a different browser to log in to your GitHub account. This can help rule out any compatibility issues with your current browser.

3. **Check for Browser Extensions:** Disable any browser extensions that might be interfering with the 2FA process, such as password managers or security extensions.

4. **Reinstall Microsoft Authenticator:** Try reinstalling the Microsoft Authenticator app on your device to ensure it's working correctly.

5. **Contact Microsoft Authenticator Support:** If the issue persists with Microsoft Authenticator, reach out to their support team directly for further assistance.

6. **Contact GitHub Support:** If none of the above steps resolve the issue, contact GitHub support directly. They can investigate the issue further and provide more specific troubleshooting steps or identify any underlying issues with your account.

In the meantime, you can still access your GitHub account from your mobile device until the issue is resolved.

**Comment by ghost** [comment]
I have already done all the steps you mentioned, I already use Microsoft Edge as a browser, I tried resetting it, I downloaded Google Chrome, the same problem continued, I tried Google Authenticator other than Microsoft authenticator, the problem is the same, I cannot solve it in any way.Is it possible to remove 2fa from my account, otherwise it looks like I will not be able to access my account08:15, 5 Aralık 2023, Dakshitha Dissanayaka ***@***.***>:
I understand that you're having trouble logging in to your GitHub account from your computer due to issues with two-factor authentication (2FA). Since mobile access is working without issues, it seems the problem is specific to your computer's setup.
Here are some steps you can try to resolve the issue:


Clear Browser Cache and Cookies: Try clearing your browser's cache and cookies to eliminate any potential conflicts or corrupted data.


Try a Different Browser: If the issue persists, try using a different browser to log in to your GitHub account. This can help rule out any compatibility issues with your current browser.


Check for Browser Extensions: Disable any browser extensions that might be interfering with the 2FA process, such as password managers or security extensions.


Reinstall Microsoft Authenticator: Try reinstalling the Microsoft Authenticator app on your device to ensure it's working correctly.


Contact Microsoft Authenticator Support: If the issue persists with Microsoft Authenticator, reach out to their support team directly for further assistance.


Contact GitHub Support: If none of the above steps resolve the issue, contact GitHub support directly. They can investigate the issue further and provide more specific troubleshooting steps or identify any underlying issues with your account.


In the meantime, you can still access your GitHub account from your mobile device until the issue is resolved.

—Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you authored the thread.Message ID: ***@***.***>
-- Yandex Mail mobil uygulamasından gönderildi

**Comment by ghost** [comment]
Please help me

**Comment by thynson** [comment]
Same issue for me. I'm using Microsoft Authenticator
I highly doubt that there are bugs on Github 2FA, at least when using with Microsoft Authenticator. When I tried to set up 2FA, it failed for multiple times to verify my code, which I double-checked that they were correct. Then it occasionally succeeded in verifying my code for once, making my account 2FA “protected”, then, it started to block me from signing in.

**Comment by ALI-WOSABY** [comment]
Same issue for me.

**Comment by samus-aran** [comment]
Hi @TurKLoJeN 

Thank you for your patience. The community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours. We appreciate your understanding! In the future, please only open 1 ticket per incident.

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion.

Thank you!

**Comment by MuddassarElexoft** [comment]
Github 2FA TOTO is not accepting any code either form MS Authenticator app OR any other chrome extension OR app.

**Reply by JuergenSimon** [reply]
Same here, total POS and EXTREMELY disruptive and annoying. If you force this crap on people, at least make sure it f*cking works.

**Comment by decatur** [comment]
MS Authenticator stopped working as 2FA. Persistent error is "Sudo authentication failed".
I had to switch to Passkeys or recovery codes.
Neither proofs "possession of my smartphone", so security just got weaker.

**Reply by Nabukodonosor** [reply]
Same here, same error.

**Comment by Nabukodonosor** [comment]
I have the same issue with MS Authenticator. Very poorly done.

**Comment by ichepurnoy** [comment]
I've had the same problem for a while: when logging-in into Github, _the notifications were arriving immediately but were incorrect_ (not accepted).  I can only speculate as to 'why'. I am from Ukraine, my phone had a firmware issued for another region (Taiwan). Previous firmware (for my region) never produced this error. The error started appearing roughly after the firmware change. Hence my thoughts: could maybe the codes be dependent on the timezone/region of the phone's firmware?

**Comment by Arghnews** [comment]
I also have the same issue with the MS Authenticator app. It's incredibly frustrating. If you're going to force people to use 2FA, could you make sure it actually works.

**Comment by ghost** [comment]
Same problem, I'm using Microsoft Authenticator. I also tried Aegis, but still same.

**Comment by mfcorgi** [comment]
I have the same issue.

**Comment by mmakarin** [comment]
Same issue with Google Authenticator.

**Comment by sirramin** [comment]
same issuse with google authenticator and 2fas auth app
very ridiculous. do something

**Comment by Iweisc** [comment]
I am having the same issue with mac os passwords app :/ i cant do anything with my github acc it. do something mf's

**Comment by rbonicel** [comment]
Not sure it's helpful, but I just discovered I have the same issue. I'm using the Authenticator extension for Firefox (moz-extension://4c4e5de9-cbf1-4d4f-b6e9-c354e7d5c384/view/licenses.html). It's been working perfectly fine for ages, and now it doesn't. I tried rescanning the QR code with the extension, but nothing. I get the "Two-factor authentification failed". Did they changed something recently ?

**Comment by Akuna444** [comment]
Make sure your phone and computer have the exactly the same time.

**Comment by yegor211** [comment]
@Akuna444 LOL! Exactly this. I disabled automatic time sync on my iPhone a while ago. And there was a 2-minute difference between PC/phone. Switching to automatic time sync immediately fixed it.

Hero! Thanks.

**Comment by kosumi32** [comment]
OMG this problem is extremely annoying, done everything as stated, scanned the QR enter the god damn code, Two factor authentication failed.

Good job

**Comment by abysslover** [comment]
I have the same problem. I am writing this comment since I am logged in github on this computer. However, I cannot log in on other machine since the codes from the Authenticator app is not working. When I reviewed my profile, I found that I have never been viewed the recovery codes. How this could happen, who has changed my recovery code as well?

**Comment by simone-romiti** [comment]
I had the same problem. In my case I solved by deleting the code on the authenticator app and adding a new one, following [these instructions](https://docs.github.com/en/authentication/securing-your-account-with-two-factor-authentication-2fa/configuring-two-factor-authentication). You need either another device where you logged in previously, or for the 1st time log in using an SMS code or a 2FA recovery code.

**Comment by ben-lewington** [comment]
I am also affected by this, using Aegis authenticator. It is affecting both my personal GH and my organisation login.

Forcing users to jump through these mandatory 2FA hoops is really only acceptable if your 2FA system actually works!

**Comment by yitecdeveloper** [comment]
Anyone :( me too now

**Comment by yuyang-ok** [comment]
I have the same issue? some extension installed on firefox and chrome.
chrome works fine and firefox not working 
microsoft Microsoft Authenticator not working too I have no idea at all

**Comment by lkimuk** [comment]
Same issue with Microsoft Authenticator. :(

**Comment by XuXinhNo1** [comment]
Same issue here. I am using Google Authenticator, when I put in the code it just said "Sudo authentication failed". I changed the MFA to the fingering passkey then when I scan my fingering, I get "Something went wrong". Pls fix it.
